ios/PodFile

```diff
@@ -23,6 +23,10 @@ target 'eMobility' do
   pod 'GoogleMaps'
   pod 'Google-Maps-iOS-Utils'
 
+  # Camera Permission
+  permissions_path = '../node_modules/react-native-permissions/ios'
+  pod 'Permission-Camera', :path => "#{permissions_path}/Camera.podspec"
```

Comment:
Are you sure you need to add it? 
It should be automatically handled by the related dependency with RN 0.6x

Reply:
Data syntax exemple:

{"tenantSubDomain":"slf", "tenantName":"SAP Labs France (prod)", "endpoint":"scp", "chargingStationID":"SAP-Mougins-18", "connectorID":2}

Reply:
Try to remove it and see if anything wrong I will push with the modification if it work without

Reply:
It's not an autolink, but a dependency for the permission camera on IOS, I've done some test on my side and I see that without this line of code the permission camera popup doesn't display on IOS

Reply:
Ok.